### PR TITLE
Changed output of ResourceGraph cmdlet to a custom class

### PR DESCRIPTION
**Pull request by fenwickt**
## Description

Changed an output of Search-AzGraph cmdlet to a custom class PSResourceGraphResponse.
Previous cmdlet output is now under Data property.
An extra property SkipToken is added so that users can leverage it for getting paginated queries.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/904
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/ResourceGraph/ResourceGraph/Models/PSResourceGraphResponse.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.ResourceGraph.Models
+{
+    using System.Collections.Generic;
+    using System.Management.Automation;
+    using Microsoft.WindowsAzure.Commands.Common.Attributes;
+
+    public class PSResourceGraphResponse
```
Comment (dramuy) [inline src/ResourceGraph/ResourceGraph/Models/PSResourceGraphResponse.cs:21]: >PSResourceGraphResponse [](http://example.com/codeflow?start=17&length=23)

now we have a response object; should we include total records and count fields as well.

**Review by dramuy — APPROVED** [review]
:shipit:

File: src/ResourceGraph/ResourceGraph/ChangeLog.md [hunk 2]
```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Output has changed to a PSResourceGraphResponse class. Previous output is now under Data property.
```
Comment (VeryEarly) [inline src/ResourceGraph/ResourceGraph/ChangeLog.md:21]:
```suggestion
* Changed output of `Search-AzGraph` to PSResourceGraphResponse which wrapped previous output under Data property.
```

**Review by VeryEarly — CHANGES_REQUESTED** [review]

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**